<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgLogin</class>
 <widget class="QDialog" name="DlgLogin">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>登陆信息</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="btnLogin">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>登陆系统</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQuit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消登陆</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbId">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>用户代码</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPwd">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>登陆密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePwd">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>89</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbUser">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">fontcolor rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbSavePwd">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>72</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
  </widget>
  <widget class="MyComBox" name="cbId" native="true">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>41</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="editable" stdset="0">
    <bool>true</bool>
   </property>
   <property name="iconSize" stdset="0">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnServer">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设置服务器</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tvServer">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>401</width>
     <height>211</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>80</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>50</number>
   </attribute>
   <column>
    <property name="text">
     <string>正在运行</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>服务器地址</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>端口</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>用户名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>数据库名</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnSelect">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>选择服务器</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加服务器</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyComBox</class>
   <extends>QWidget</extends>
   <header>mycombox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
